casadm/statistics_model.c

```diff
@@ -628,6 +628,10 @@ static int cache_stats(int ctrl_fd, const struct kcas_cache_info *cache_info,
 	if (stats_filters & STATS_FILTER_CONF)
 		cache_stats_conf(ctrl_fd, cache_info, cache_id, outfile, by_id_path);
 
+	/* Don't print stats for a cache in standby state */
+	if(cache_info->info.state & (1 << ocf_cache_state_standby))
```

Comment:
Add a space after if
Was blocking stats intended for standby cache? I remember some discussion, that we wanted to display at least some values (but it might have changed since then)